### ISSUE TITLE
Succeeded in reproducing the result on Attribute Prediction (not tested the landmark detection performance)

### ISSUE THREAD

**Issue by nashory** [fdjingyuan/Deep-Fashion-Analysis-ECCV2018#12]
@pbamotra, @xuanle22, @ZhuXiang-tuxing
Hi, I ran the code to confirm if the performance is reproduces properly.
Although I customized the metric functions to output the f1 scores together, but the result should be reproducible as it is. (I failed to reproduce the "style" result. I can't understand how the authors achieved 68.82 recall@3 in their paper. Is there any thicks exist not mentioned in their paper?)
I trained the model with the default parameters that the authors provided.
The recall@3/5 is measured when the model shows the best f1@1 score during the training.
The below is the result table.

Recall@3/5 | Category | Texture | Fabric | Shape | Part | Style | All
-- | -- | -- | -- | -- | -- | -- | --
batch=32 / lr= 0.0001 | 91.18 / 95.82 | 56.85 / 66.34 | 43.35 / 53.66 | 57.99 / 66.49 | 43.26 / 53.21 | 32.54 / 40.87 | 57.41 / 65.77
batch=32 * 8 / lr= 0.0001 * 8 | 89.42 / 94.77 | 52.08 / 61.40 | 38.22 / 48.38 | 54.54 / 63.41 | 39.2 / 48.42 | 27.88 / 37.27 | 53.56 / 62.07

I found that when I trained the model with Multiple GPUs (8) with 8 times of learning rate, the performance degrades severely. Pleas make sure that you find the correct learning rate and batch size when training your model.

I attached the evaluation curve of 1-GPU (lr=0.0001, batch=32) vs. 8-GPU (lr=0.0001 x 8, batch=32 x 8)
The blue one is for the 1-GPU.

<img width="1180" alt="스크린샷 2019-12-29 오후 1 11 48" src="https://user-images.githubusercontent.com/17468992/71552265-d77e3b80-2a3c-11ea-9e4c-c32e3c631c9b.png">

**Comment by ashwath98** [comment]
Hey. Could I know your system specs?[Pytorch version etc]

**Comment by ousinkou** [comment]
My reproduced result is nearly same as you, the style recall result is weird

**Comment by nashory** [comment]
@ousinkou I ended up with not trusting the "style" score, please refer to my paper accepted to ECCV 2020.
https://arxiv.org/abs/2007.06769
![스크린샷 2020-12-20 오후 8 43 20](https://user-images.githubusercontent.com/17468992/102712386-53375f00-4304-11eb-943b-5302fcd93e23.png)

**Comment by ousinkou** [comment]
@nashory Well, thanks for your reply. Style scores were not reproducible using publicly released code，that's sad.

**Comment by young-yangb** [comment]
@nashory Hi, I have a question to consult you. Before we train the landmark branch and the category/attribute prediction network jointly, are we supposed to train the landmark branch solely and use its weight in category/attribute network？Any reply will be highly appreciated.

**Comment by young-yangb** [comment]
@ousinkou Hi, I have a question to consult you. Before we train the landmark branch and the category/attribute prediction network jointly, are we supposed to train the landmark branch solely and use its weight in category/attribute network？Any reply will be highly appreciated.

**Comment by zhushumin** [comment]
Hi, I have a question. How do you get your ALL@top3 58.02 and ALL@top5 64.35,   I have got the similar results as you on the five attributes groups. But, my ALL@top3 and ALL@top5 is very low.

**Comment by zhushumin** [comment]
Any prediction code is very appreciate.

**Comment by abhigoku10** [comment]
@nashory  can you please share the pretrained weigth file and inference code it would be helpful
